app/src/ui/changes/commit-message.tsx

```diff
@@ -212,11 +214,18 @@ export class CommitMessage extends React.Component<
 
   private canCommit(): boolean {
     return (
-      (this.props.anyFilesSelected && this.state.summary.length > 0) ||
-      this.props.singleFileCommit
+      !this.props.anyConflictedFiles &&
+      ((this.props.anyFilesSelected && this.state.summary.length > 0) ||
+        this.props.singleFileCommit)
     )
   }
 
+  private commitButtonTooltipText() {
+    return hasConflictedFiles
```

Comment:
Since this is a function, wouldn’t this always be truthy?

Reply:
good catch!!